packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -987,13 +1027,18 @@ export default class ScheduledPaymentModule {
       recurringDayOfMonth,
       recurringUntil
     );
+    options.listener?.onEndSpHashCreation?.();
```

Comment:
Should this have the hash as an argument?

Reply:
It could but we have no immediate use for it.